src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -72,6 +73,9 @@ object Boot extends App with LazyLogging {
 
     for {
       _ <- schemaDAO.init() recover {
+        case e: WorkbenchException =>
```

Comment:
was thinking of making a broad.io link and having it link to some confluence page with instructions